### ISSUE TITLE
Success Criterion 2.5.5 Target Size Failure

### ISSUE THREAD

**Issue by codyduong** [codyduong/codyduong.dev#33]
> The size of the [target](https://www.w3.org/TR/WCAG21/#dfn-target) for [pointer inputs](https://www.w3.org/TR/WCAG21/#dfn-pointer-inputs) is at least 44 by 44 [CSS pixels](https://www.w3.org/TR/WCAG21/#dfn-css-pixels) except when:

![image](https://user-images.githubusercontent.com/24193268/217417884-8321f260-2b27-439c-95b1-8584d54839cc.png)
Each icon here in Navbar is only 32x32px

![image](https://user-images.githubusercontent.com/24193268/217417958-21ebab95-4ed2-4009-be5c-3e9f3f3496f9.png)
Each icon here in Footer is only 28x28px

**Comment by codyduong** [comment]
Increasing the size is disgusting, instead add navigation links to the footer
